metashare/asgi.py

```diff
@@ -6,14 +6,22 @@
 import os
 
 import django
+import sentry_sdk
 from channels.routing import get_default_application
 from newrelic import agent
+from sentry_sdk.integrations.asgi import SentryAsgiMiddleware
 
 agent.initialize()
 agent.wrap_web_transaction("django.core.handlers.base", "BaseHandler.get_response")
 
 os.environ.setdefault("DJANGO_SETTINGS_MODULE", "config.settings")
+SENTRY_DSN = os.environ.get("SENTRY_DSN")
+if SENTRY_DSN:  # pragma: nocover
+    sentry_sdk.init(dsn=SENTRY_DSN)
```

Comment:
@wlonk I would've assumed we could skip this here since we're already doing it in `base.py`?

Reply:
Maybe; I wasn't confident that would be run in time, though. But I think it is because we have the `django.setup()` call there? We could try removing this.